src/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__item {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 > .stars__item:nth-child(-n + 4) {
```

Comment:
.stars--3 should work too )

<img width="477" alt="Screenshot 2022-07-19 at 11 56 37" src="https://user-images.githubusercontent.com/47606379/179710454-be61e40f-5ba9-4988-be52-d34a26d0b415.png">